### ISSUE TITLE
Smartbanner position inside body

### ISSUE THREAD

**Issue by sebastianhelbig** [ain/smartbanner.js#120]
**Currently** the smartbanner div is appened with `append`. In SPA applications where the DOM is created by JavaScript this leads to the problem that the position inside the body is kind of random. I.e. when using the SPA framework Ember, one of the two results will occur "randomly":

```
<body>
  <div class="smartbanner">…</div>
  <div class="ember-view">…</div>

<body>
  <div class="ember-view">…</div>
  <div class="smartbanner">…</div>
```

This makes styling via CSS unnecessarily hard.

**It would be good** if the smartbanner div gets always appended as the first child inside body.

**Comment by ain** [comment]
Thanks for the heads up!

For Single-page Applications the most obvious recommendation would be to go with the API-based solution that was started in #103, but is struggling to be signed off. You're welcome to contribute!

**Comment by ain** [comment]
API has now been released in [v1.14.0](https://github.com/ain/smartbanner.js/releases/tag/v1.14.0). This should allow you to avoid race conditions by being able to append at the right point of time.
